staging/src/kubevirt.io/client-go/api/v1/register.go

```diff
@@ -0,0 +1,141 @@
+/*
```

Comment:
everything this PR is introducing under`staging/src/kubevirt.io/client-go/api` needs to go